packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -122,13 +123,15 @@ interface BaseMongoSelectProps {
 interface ProjectMongoSelectProps extends BaseMongoSelectProps {
   data?: Array<ProjectInterface>;
   current?: CurrentProjectInterface;
-  constructProjectURL: (orgID: string, projectID: string) => string;
+  constructProjectURL: NonNullable<MongoNavInterface['constructProjectURL']>;
```

Comment:
I've tried using NonNullable before and didn't get it to work, nice to have this in the repo now!